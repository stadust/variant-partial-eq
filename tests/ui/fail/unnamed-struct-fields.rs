use variant_partial_eq::VariantPartialEq;

#[derive(VariantPartialEq)]
struct Unnamed(String, u32);

fn main() {}