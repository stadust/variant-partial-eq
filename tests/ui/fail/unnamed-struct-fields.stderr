error: #[derive(VariantPartialEq)] only supports structs with names fields
 --> tests/ui/fail/unnamed-struct-fields.rs:4:15
  |
4 | struct Unnamed(String, u32);
  |               ^^^^^^^^^^^^^
